tests/dummy/app/templates/docs/guides/upgrading.md

```diff
@@ -0,0 +1,126 @@
+# Upgrading from ember-electron 2.x
+
+While upgrading from 2.x to 3.x should not be very time-consuming in most cases, quite a bit has changed, so you should follow these steps in order to be successful. Starting from a clean tree (so you can run a clean `git diff` afterwards to see what changed), the recommended steps are:
+
+## Upgrade ember electron
+
+```sh
+yarn-or-npm remove ember-electron
+ember install ember-electron
+```
+
+When prompted to overwrite `testem-electron.js`, choose `yes`.
+
+## Fix electron version
+
+Your `electron-app` project will initially have the most recent release of Electron installed. You were probably running an older version since it was very difficult to run newer than Electron 4.x with `ember-electron` 2.x. So if you don't want to deal with upgrading Electron in the middle of upgrading `ember-electron`, you should change your `electron-app/package.json`'s `electron` version to match the version of `electron-prebuilt-compile` (which would determine the Electron version you were running) that is in your root `package.json`. Don't forget to run `yarn`/`npm install` from `electron-app` afterwards!
+
+The good news is that now upgrading Electron just involves updating the version installed in the `electron-app` package -- no more waiting for `electron-prebuilt-compile` to release an update matching the latest Electron version!
```

Comment:
```suggestion
The good news is that now upgrading Electron only involves updating the version installed in the `electron-app` package -- no more waiting for `electron-prebuilt-compile` to release an update matching the latest Electron version!
```

Reply:
(same comment here as above)

Reply:
I'm not the one who started the war on "just", I know it gets flagged by a lot of tools as being discriminatory or something though.

Reply:
Like is it an ESL thing? Can you link to any articles or anything?

To be blunt, I'm not inclined to follow advice that seems wrong to me if I don't have any understanding of the reasoning behind it. I'm certainly not going to die on this hill if you feel really strongly about it, but even after a little googling, I'm unable to get any sense of why `only` would be preferable to `just` here.

Reply:
I think the argument is it has a negative connotation like "just do it, it's easy", "just make it work" etc. This case doesn't necessarily apply. I don't particularly care that much, only following what I have heard. There was a talk at EmberConf about it, and it sounds like whatever tool they use only looks for how many times you said "just", "guys", etc anything that could be referring to gender or implying that something is easy, when it is not easy for everyone.